entrypoint.sh

```diff
@@ -1,21 +1,22 @@
 #!/usr/bin/env bash
-
-set -euo pipefail
-
+set -ev
 cd "$GITHUB_WORKSPACE"
+if [[ -z "$INPUT_NAME" ]]; then
+    INPUT_NAME=$(basename "$GITHUB_REPOSITORY")
+fi
+INPUT_DIRECTORY=$(realpath "$PWD/$INPUT_DIRECTORY")
 
-# Absolutely no idea why GitHub Actions seems convinced this package isn't
-# installed already by this point, but whatever.
-raco pkg install --batch --auto https://github.com/jackfirth/resyntax.git
+git fetch --depth=1 origin "$GITHUB_BASE_REF"
 
-if [ "$INPUT_DIRECTORY" == "." ]; then
-  # As of August 2019, installing a linked package with source directory "."
-  # raises an error for some bizarre reason, even when the package's name is
-  # specified explicitly with --name. This is especially strange because source
-  # directories like "./foo" work just fine.
-  raco pkg install --name "$INPUT_NAME" --batch --auto --link
-else
-  raco pkg install --name "$INPUT_NAME" --batch --auto --link "$INPUT_DIRECTORY"
-fi
+GITHUB_BASE_REF="origin/$GITHUB_BASE_REF"
```

Comment:
Is `GITHUB_BASE_REF` used for anything after this line?

Reply:
Yeah, by the `racket-package-resyntax-action` script.